modules/rostests/apitests/crt/strlen.c

```diff
@@ -48,7 +48,12 @@ Test_strlen(PFN_STRLEN pstrlen)
     eflags = __readeflags();
     __writeeflags(eflags | EFLAGS_DF);
     len = pstrlen(teststr + 4);
-    ok((__readeflags() & EFLAGS_DF) != 0, "Direction flag in ELFAGS was changed.");
+
+#ifdef _M_AMD64
+    ok((__readeflags() & EFLAGS_DF) != 0, "Direction flag in ELFAGS was changed.\n");
```

Comment:
The string should read "EFLAGS", not "ELFAGS" (and same in the other one below).